.circleci/config.yml

```diff
@@ -186,13 +216,28 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - build-and-test:
+      - build:
+          filters:
+            tags:
+              only: /.*/
+      - test:
           filters:
             tags:
               only: /.*/
+          requires:
+            - build
+      - contile-integration-tests:
+          name: contile-integration-tests
+          requires:
+          - build
+          filters:
+            branches:
+              only:
```

Comment:
Sounds like this won't add much time to CI so let's run this everywhere like the other targets. It'll be nice to have it catch issues on pull requests.

Reply:
okay. Made the changes. Thanks for the r+!